### ISSUE TITLE
Use HSL based deploy script to deploy digitransit to AKS

### ISSUE THREAD

**Issue by nandor-poka** [mfdz/digitransit-ui#9]
The official project has deploy scripts that can / could be  used out of the box. Look up the repo for deploy scripts to find the  right one and run it. Implement modifications if necessary. If need Estimation: 48h

**Comment by frankgerhardt** [comment]
Use digitransit-kubernetes-deploy

**Comment by Beck-berry** [comment]
[Here they used Kubernetes for the Ulm Digitransit project.](https://github.com/verschwoerhaus/digitransit-kubernetes) It is only one long YAML file without documentation but it is quite readable so I guess we shall exchange their docker images and URLs to ours.. Or to the original to test whether it works at all.

**Comment by frankgerhardt** [comment]
re: runs fine. Does it mean you can open the frontend in the browser and see something from the backend?

**Comment by meromisi** [comment]
I tried to deploy it on AKS, first attempt failed. Multiple pods were terminated (not running). In the YML file there is section for the secret it uses with the secret.yml.example file and it was there but commented (like # and so on). I removed the comment marks(#) and after it had only one error, but it was a major error,  the pod for the digitransit-ui-ulm wasn't running.  Couldn't solve the problem so far...
